qan-api2/.codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
```

Comment:
That file can be removed, and qan should be added to the `codecov.yml` in the root folder